src/v2/Apps/Conversation/Components/Reply.tsx

```diff
@@ -53,7 +53,7 @@ export const Reply: React.FC<ReplyProps> = props => {
   const textArea = useRef()
 
   return (
-    <StyledFlex p={1} right={[0, null, null, null, "376px"]}>
+    <StyledFlex p={1} right={[0, null, null, null, null]}>
```

Comment:
Now that the detail box is collapsible on `xl` screens, the reply box needs to take the  whole remaining width.

Reply:
```suggestion
    <StyledFlex p={1} right={[0, null]}>
```

Given that the only difference here is the xs breakpoint, you can just provide two values and the latter value will be used for all the higher breakpoints.